core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessorState.java

```diff
@@ -584,6 +596,9 @@ private void doQueryRecord(HttpChunkedResponseSocket socket, int columnCount) {
                     break;
                 case ColumnType.LONG128:
                     throw new UnsupportedOperationException();
```

Comment:
i appreciate this is scope widening, but can i propose we print LONG128 as decimal?

Reply:
this will be done in a separate PR as discussed.